pkg/domain/entities/play.go

```diff
@@ -54,6 +55,9 @@ type PlayKubeOptions struct {
 	LogOptions []string
 	// Start - don't start the pod if false
 	Start types.OptionalBool
+	// Body is a reader to read the body of the request from.
+	// Overrides the "path" variable if set.
+	Body io.Reader
```

Comment:
Why do you change this in `pkg/domain/entities`?
This should only be used by podman, external users should only use `pkg/bindings` so I do not see why this is needed.

Reply:
That's fine, there are some cases where I want to compile in libpod :)

Reply:
If you want me to exclude that portion of the commit I can, but regardless I'll have to keep a separate fork to add the param.

Reply:
I don't want to add options that we do not use, if you want to simplify this why even use a path in the first place for this function. We could open the file in cmd/podman and pass this down as io.Reader to PlayKube instead of the path.

But maybe other maintainers have a different opinion here.

Reply:
I agree, should force passing the io.Reader, no path.

Reply:
I was going for the approach of making as minimal of changes as possible, but if you want me to refactor it that way, can do.

Maybe best to just merge the bindings/ changes first and later the others?

Reply:
OK - I have refactored everything to use a io.Reader instead of a path.

This is significantly cleaner in the HTTP handlers, which previously were reading the body to a tempfile on disk. Now they can pass the http body directly to PlayKube and remove the tempfile altogether.